src/features/ui/WrappedDropdownMenu/WrappedDropdownMenu.tsx

```diff
@@ -28,25 +27,21 @@ export const WrappedDropdownMenu: FC<WrappedDropdownMenuProps> = ({
 	hasError,
 	helperText,
 }) => {
-	const trigger = (
-		<Input
-			type="text"
-			placeholder={placeholder}
-			value={value}
-			onChange={() => {}}
-			error={hasError}
-			helperText={helperText}
-			endEnhancer={<IconChevronDown />}
-		/>
-	);
-
 	return (
 		<InputWrapper
 			className={classNames(styles.Container, className)}
 			label={label}
 			info={info}
```

Comment:
Hey Ash, this might be a little out of scope for task - could we rename `info` ? At first glance I'm unsure of what `info` would be for which suggests a change of name to be something more specific?

Reply:
Good call! Have updated in all instances to `infoTooltipText`.